apps/web/src/styles/_shame.scss

```diff
@@ -1,4 +1,4 @@
 .pins-content--sticky {
```

Comment:
this should be on 9-pages folder on the affected page file

Reply:
this was here already ?

Reply:
leave it then